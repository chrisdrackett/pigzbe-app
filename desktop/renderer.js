import {AppRegistry} from 'react-native';
import App from './bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
