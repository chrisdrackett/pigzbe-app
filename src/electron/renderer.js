import {AppRegistry} from 'react-native';
// import App from '../../App';
import App from '../../dist/bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
