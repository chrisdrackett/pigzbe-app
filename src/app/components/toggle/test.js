import React from 'react';
import renderer from 'react-test-renderer';
import Toggle from './';

const props = {
    label: 'Label',
    onPress: () => {},
    active: false,
};

describe('Toggle', () => {
    test('renders correctly', () => {
        const tree = renderer.create(<Toggle {...props}/>).toJSON();
        expect(tree).toMatchSnapshot();
    });

    test('renders correctly in plain mode', () => {
        const tree = renderer.create(
            <Toggle {...Object.assign({}, props, {
                plain: true
            })}/>
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
