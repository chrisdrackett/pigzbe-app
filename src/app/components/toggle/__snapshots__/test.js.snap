// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "alignSelf": "stretch",
    }
  }
>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "rgb(21, 74, 150)",
        "borderColor": "rgb(21, 74, 150)",
        "borderRadius": 22.5,
        "borderWidth": 1,
        "marginBottom": 10,
        "maxWidth": 400,
        "opacity": 1,
        "width": "100%",
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "alignSelf": "stretch",
            "color": "rgb(255, 255, 255)",
            "fontFamily": "Poppins",
            "fontSize": 14,
            "fontWeight": "bold",
            "height": 45,
            "paddingLeft": 20,
            "paddingRight": 20,
            "paddingTop": 12,
            "textAlign": "center",
          },
          undefined,
          null,
          undefined,
        ]
      }
    >
      Label
    </Text>
  </View>
</View>
`;

exports[`Button renders correctly in plain mode 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "alignSelf": "stretch",
    }
  }
>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "rgb(21, 74, 150)",
        "borderColor": "rgb(21, 74, 150)",
        "borderRadius": 22.5,
        "borderWidth": 1,
        "marginBottom": 10,
        "maxWidth": 400,
        "opacity": 1,
        "width": "100%",
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "alignSelf": "stretch",
            "color": "rgb(255, 255, 255)",
            "fontFamily": "Poppins",
            "fontSize": 14,
            "fontWeight": "bold",
            "height": 45,
            "paddingLeft": 20,
            "paddingRight": 20,
            "paddingTop": 12,
            "textAlign": "center",
          },
          undefined,
          null,
          undefined,
        ]
      }
    >
      Label
    </Text>
  </View>
</View>
`;

exports[`Toggle renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "rgb(255, 255, 255)",
        "color": "rgb(0, 50, 120)",
      },
      undefined,
    ]
  }
>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "rgb(255, 255, 255)",
        "borderColor": "rgb(21, 74, 150)",
        "borderRadius": 22.5,
        "borderWidth": 1,
        "marginBottom": 20,
        "maxWidth": 400,
        "opacity": 1,
        "width": "100%",
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "alignSelf": "center",
            "color": "rgb(0, 50, 120)",
            "fontFamily": "Poppins",
            "fontSize": 14,
            "fontWeight": "bold",
            "textAlign": "center",
          },
          undefined,
          null,
          undefined,
        ]
      }
    >
      Label
    </Text>
  </View>
</View>
`;

exports[`Toggle renders correctly in plain mode 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "rgb(255, 255, 255)",
        "color": "rgb(0, 50, 120)",
      },
      undefined,
    ]
  }
>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "rgb(255, 255, 255)",
        "borderColor": "rgb(21, 74, 150)",
        "borderRadius": 22.5,
        "borderWidth": 1,
        "marginBottom": 20,
        "maxWidth": 400,
        "opacity": 1,
        "width": "100%",
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "alignSelf": "center",
            "color": "rgb(0, 50, 120)",
            "fontFamily": "Poppins",
            "fontSize": 14,
            "fontWeight": "bold",
            "textAlign": "center",
          },
          undefined,
          null,
          undefined,
        ]
      }
    >
      Label
    </Text>
  </View>
</View>
`;
